.github/workflows/scenario-test.yaml

```
@@ -1,106 +1,95 @@
 name: scenario-test
 on:
+  schedule:
+    - cron: 0 12 * * *
   workflow_dispatch:
-
+  pull_request:
+    paths:
+      - simulator*.repos
+      - .github/workflows/scenario-test.yaml
 jobs:
   scenario-test:
-    runs-on: [self-hosted, Linux, X64]
-    container: ghcr.io/autowarefoundation/autoware:universe-devel
+    runs-on: ubuntu-24.04
+    container:
+      image: ghcr.io/autowarefoundation/autoware:universe-devel
     steps:
       - name: Check out repository
         uses: actions/checkout@v4
-
-      - name: Show disk space before the tasks
-        run: df -h
-
-      - name: Show machine specs
-        run: lscpu && free -h
-
       - name: Install required packages
+        shell: bash
         run: |
           apt-get update
           apt-get install -y \
             unzip \
+            zip \
             curl \
             gnupg2 \
             lsb-release \
             python3-pip \
             git \
             ros-humble-pcl-ros
           pip install --upgrade gdown vcstool
+      - name: Show memory info
         shell: bash
-
+        run: free -h
       - name: Clone Autoware and Import simulator.repos
+        shell: bash
         run: |
           mkdir -p ~/autoware_ws/src
           cd ~/autoware_ws
           git clone --recurse-submodules https://github.com/autowarefoundation/autoware.git src/autoware
           cd src
           vcs import < autoware/simulator.repos
-        shell: bash
-
       - name: Install ROS dependencies
+        shell: bash
         run: |
           cd ~/autoware_ws
           rosdep update
```

It's probably better to source the ROS/Autoware workspace before running rosdep

```suggestion
          source /opt/ros/humble/setup.bash
          source /opt/autoware/setup.bash
          rosdep update
```

---

thank you it done